src/main/java/com/blamejared/contenttweaker/fluids/FluidBuilder.java

```diff
@@ -137,4 +155,8 @@ public void build(ResourceLocation location) {
         VanillaFactory.queueFluidForRegistration(flowingFluid);
         VanillaFactory.queueItemForRegistration(bucketItem);
     }
+
+    private static int setTotallyOpaqueWhenNotDefineAlphaChannel(int color) {
```

Comment:
```suggestion
    private static int opaqueUnlessSpecified(int color) {
```

Personally think that is a better name.